src/mme-app/main.cpp

```diff
@@ -27,6 +27,8 @@
 #include <utils/mmeTimerUtils.h>
 #include "mmeStatsPromClient.h"
 
+#define S10_FEATURE
```

Comment:
Move all occurrence of S10_FEATURE and keep it in Makefile.

Reply:
Yes, sure.